src/db.c

```diff
@@ -1688,6 +1688,11 @@ int genericGetKeys(int storeKeyOfs, int keyCountOfs, int firstKeyOfs, int keySte
     return result->numkeys;
 }
 
+int sinterGetKeys(struct redisCommand *cmd, robj **argv, int argc, getKeysResult *result) {
```

Comment:
This is the syntax for SINTERCARD (SINTER has a completely different one)
```suggestion
int sintercardGetKeys(struct redisCommand *cmd, robj **argv, int argc, getKeysResult *result) {
```